misc-utils/fincore.c

```diff
@@ -19,8 +19,10 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include <sys/ioctl.h>
```

Comment:
Are these header additions still needed?

Reply:
thank you for the catch, I think they are no longer needed now.